.travis.yml

```diff
@@ -32,7 +32,7 @@ env:
   - TEST_CLUSTER=minikube
   - TEST_KUBECTL_VERSION=v1.15.0
   - TEST_NSENTER_VERSION=2.32
-  - TEST_HELM_VERSION=v2.9.1
+  - TEST_HELM_VERSION=v3.0.2
```

Comment:
While the Helm 2 charts are still being kept, I think we should still keep the tests for them, so it'd be better to introduce a new env var for testing Helm 3 (and avoid any possible breaking changes by relying on testing Helm 2 charts using the v3 CLI).